uint8 role
uint8 id
uint8 mix
uint8 group
LinktrackSetting[] nodes
